etcd.py

```diff
@@ -241,8 +243,9 @@ def delete_member(self, member):
         self.adjust_security_groups('revoke_ingress', member)
         return result
 
-    def etcd_arguments(self, data_dir, initial_cluster, cluster_state):
-        return [
+    def etcd_arguments(self, data_dir, initial_cluster, cluster_state, run_old)):
```

Comment:
```python
-    def etcd_arguments(self, data_dir, initial_cluster, cluster_state, run_old)):
+    def etcd_arguments(self, data_dir, initial_cluster, cluster_state, run_old):
```

this is the reason why tests are failing

Reply:
damn, sorry about that.